packages/design-system-react/src/components/AvatarFavicon/AvatarFavicon.stories.tsx

```diff
@@ -64,29 +63,29 @@ type Story = StoryObj<typeof AvatarFavicon>;
 
 export const Default: Story = {
   args: {
-    src: SAMPLE_AVATARFAVICON_URIS[0],
-    name: 'Adobe',
-    fallbackText: 'OS',
+    src: '/images/favicons/metamask.svg',
+    name: 'MetaMask Portfolio',
+    fallbackText: 'MP',
   },
 };
 
 export const Src: Story = {
   render: () => (
     <div className="flex gap-2">
       <AvatarFavicon
-        name="Adobe"
-        fallbackText="A"
-        src={SAMPLE_AVATARFAVICON_URIS[0]}
+        name="MetaMask Portfolio"
+        fallbackText="MM"
+        src="/images/favicons/metamask.svg"
```

Comment:
- The direct path immediately shows which image is being used
- No need to look up what SAMPLE_AVATARFAVICON_URIS[0] maps to
- Self-documenting code - the path tells you exactly what the image is
- Reduces cognitive load - no need to map array indices to actual images

Reply:
SAMPLE_AVATARFAVICON_URIS is used in both AvatarFavicon and AvatarGroup. This helps keeping all the references in 1 place

Reply:
cons: not reusable in favicon-related components

Reply:
It looks like these constants are also being bundled into the build folder. While I don’t see the `assets/` folder there, the constants may not be tree-shaken out. If we want to keep these types of constants, we should move them to a separate file. See [this comment](https://github.com/MetaMask/metamask-design-system/pull/555#discussion_r2049719567)